kuma/users/templates/users/profile.html

```diff
@@ -94,6 +96,13 @@ <h1 class="page-title">
           {% endif %}
       </div>
 
+      {% if profile.is_banned and request.user.is_superuser %}
+      <div class="banned">
+          User banned {{ datetimeformat(profile.active_ban().date, format='date') }} 
+          by {{ profile.active_ban().by }}.
+      </div> 
+      {% endif %}
```

Comment:
I'm not convinced. I don't think they belong in the same place just because they both relate to bans. I think textual information about the profile is lined up on the left under the username, and actions to take on the profile are in buttons on the right. 

This approach has the added benefit of not requiring UI experimentation to try and cram textual data in a space normally occupied by a button. 

I nominate @stephaniehobson to break the tie.